README.md

```diff
@@ -44,6 +50,37 @@ You will first need to setup your local environment and ensure that all configur
 
 ## <a name="contributing-your-code"></a> Contributing Your Code
 
+Are you willing to contribute to this project? You can contribute in many areas but primarily in the following areas
+
+1. Implementing endpoints and controllers
+2. Writing unit tests for endpoints and controllers. 
+3. Documentation 
+4. Creating middleware and their consumables
+5. Fixing/pointing out bugs
+6. We could use a boost in our code coverage, so any tests to cover untested fucntions is highly welcome
+
+### Quick Reference
+
+- All responses must follow the format specified in the online [swagger documentation](https://comments-microservice.herokuapp.com/). To facilitate this a few utility functions have been provided and you should use them to send your responses to stay consistent. 
+   + **responseHandler** for sending responses with custom response code, optional data and a message 
+   + **customError** for error responses with custom error response code, message and optional data. To use this in a try..catch block, create a new instance of the customError class with the above properties. Pass this new error object to the `next` callback in your controller znd that is it. An **errorHandler** utility will send the necessary response
+
+- A validation middleware is created to validate all endpoints using the `Joi` validation package. Validation rules are available for every endpoint and you must use a validation middleware when implementing an endpoint. If the validation rule/middleware for the endpoint you are not working on has not been implemented then contact the maintainers. 
+
+- Some endpoints are guarded and require authenication via JWT tokens. These endpoints are indicated in the swagger documentation. Authentication middleware have been provided for these endpoints. Make use of appropriate auth middleware, where required, when implementing these endpoints.
+
+- To enable this project move swiftly. In the initial stages, tests were not written alongside implementation. For this reason, tests and implementation are separate. To enable tests not fail for non-exiatent endpoints, a helper method, **describeIfEndpoint**, has been created to defer running of tests until the endpoint itself has been created. Please use when necessary. When writing tests, for a minimum, test for a valid request's response, auth error response (if auth required), and validation error response.
+
+- If for any reason, your contribution is impeded because some component mentioned above is not available, kindly reach out to the maintainers so that you will be advised on how to proceed.
+
+- Always lint your code using `npm run lint:fix` to catch and fix any sysntax errors, format your code with our code style choice, `prettier`. Do this also before pushing your commits to a new or an existing PR. If your code fails linting, it will not be accepted.
```

Comment:
This paragraph is redundant as there is already a section called "Linting Your Code"